config/semi_automated.yml

```diff
@@ -449,6 +453,11 @@ parameters:
             tag: 'OAuth CID: 860'
             link: User:Enterprisey/YABBR
             namespaces: [0]
+    en.wikisource.org:
+        Change status:
+            link: User:DannyS712/Change_status
+            regex: \[\[User:DannyS712/Change status\|Change status\.js\]\]
+            namespaces: [104]
```

Comment:
I'm OK to merge this because en.wikisource's config is already so small, but in general we shouldn't be adding scripts to AutoEdits that have very low usage. Each and every script added slows down the tool a tad bit more.

Reply:
Understandable - I figured because of the few scripts already given, as well as the namespace restriction, it wouldn't be too much of an issue